1.0/test/imagestream.json

```diff
@@ -3,5 +3,20 @@
   "apiVersion": "v1",
   "metadata": {
     "name": "wildflyswarm-10-centos7"
-  }
+  },
```

Comment:
Should we remove the build-config.json and imagestream.json since they are now deprecated by the new wildflyswarm-sti-all.json?